packages/rpc-types/src/transaction.ts

```diff
@@ -144,7 +144,7 @@ type TransactionForFullMetaBase = Readonly<{
     status: TransactionStatus;
 }>;
 
-type TransactionForFullMetaInnerInstructionsUnparsed = Readonly<{
+export type TransactionForFullMetaInnerInstructionsUnparsed = Readonly<{
```

Comment:
@mcintyre94, is there a reason we don't use these in `getTransaction`? They're copy/pasted in, and I feel like there was a reason for that?

Reply:
I'm not sure, but my guess would be that it was before we untangled the rpc packages and it would have caused a circular dependency mess. Probably can be changed now!